api.rst

```diff
@@ -0,0 +1,8 @@
+
+.. If you created a package, create one automodule per module in the package.
+
+.. automodule:: onewire_bus
+   :members:
+
+.. automodule:: onewire_device
```

Comment:
I think these will need to be `adafruit_onewire.bus` and `adafruit_onewire.device`.

Reply:
thanks. nice catch. left over from a rename.